src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/NewAzureSqlFailoverGroup.cs

```diff
@@ -30,62 +30,73 @@ namespace Microsoft.Azure.Commands.Sql.FailoverGroup.Cmdlet
     {
         /// <summary>
+        /// Gets or sets the name of the server to use.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1,
+            HelpMessage = "The name of the primary Azure SQL Database Server of the Failover Group.")]
+        [ValidateNotNullOrEmpty]
+        public string ServerName { get; set; }
+
+        /// <summary>
         /// Gets or sets the name of the Failover Group to create.
         /// </summary>
         [Parameter(Mandatory = true,
-            HelpMessage = "The name of the Azure SQL FailoverGroup to create.")]
+            HelpMessage = "The name of the Azure SQL Database Failover Group to create.")]
         [ValidateNotNullOrEmpty]
         public string FailoverGroupName { get; set; }
 
         /// <summary>
         /// Gets or sets the partner resource group name for Azure SQL Database Failover Group
         /// </summary>
         [Parameter(Mandatory = false,
-            HelpMessage = "The partner resource group name for Azure SQL Database Failover Group.")]
+            HelpMessage = "The name of the secondary resource group of the Azure SQL Database Failover Group.")]
         [ValidateNotNullOrEmpty]
         public string PartnerResourceGroupName { get; set; }
 
         /// <summary>
         /// Gets or sets the partner server name for Azure SQL Database Failover Group
         /// </summary>
         [Parameter(Mandatory = true,
-            HelpMessage = "The partner server name for Azure SQL Database Failover Group.")]
+            HelpMessage = "The name of the secondary server of the Azure SQL Database Failover Group.")]
         [ValidateNotNullOrEmpty]
         public string PartnerServerName { get; set; }
 
         /// <summary>
         /// Gets or sets the failover policy without data loss for the Sql Azure Failover Group.
         /// </summary>
         [Parameter(Mandatory = false,
-            HelpMessage = "The failover policy without data loss for the failover group.")]
+            HelpMessage = "The failover policy of the Azure SQL Database Failover Group.")]
         [ValidateNotNullOrEmpty]
         public FailoverPolicy FailoverPolicy { get; set; }
 
         /// <summary>
         /// Gets or sets the grace period with data loss for the Sql Azure Failover Group.
         /// </summary>
         [Parameter(Mandatory = false,
-            HelpMessage = "The window of grace period that we tolerate with data loss during a failover operation for the failover group.")]
+            HelpMessage = "The grace period during outage before automatic failover with data loss of the Failover Group is triggered.")]
         [ValidateNotNullOrEmpty]
-        [Obsolete("This parameter will be deprecated in the next release.")]
-        public int GracePeriodWithDataLossHour { get; set; }
+        [ValidateRange(0, int.MaxValue)]
+        public int GracePeriodWithDataLossHours { get; set; }
```

Comment:
@dlemMSFT @akromm if we're renaming this cmdlet, what do you guys think about renaming it to `GracePeriodWithDataLossInHours`? 

I say this because I know StaticAnalysis is going to throw an exception because parameter nouns should be singular (but nouns such as `InHours`, `InMinutes`, etc. are suppressed and won't trigger an exception). Also, I think `InHours` sounds better with the rest of the parameter noun than `Hours`. Let me know what you think.